code/game/objects/items/devices/holowarrant.dm

```diff
@@ -44,57 +45,81 @@
 	M.examinate(src)
 
 //sync with database
-/obj/item/device/holowarrant/verb/sync(mob/living/carbon/user as mob)
+/obj/item/device/holowarrant/proc/sync()
 	if(!isnull(data_core.general))
 		for(var/datum/data/record/warrant/W in data_core.warrants)
 			storedwarrant += W.fields["namewarrant"]
 		to_chat(usr, "<span class='notice'>The device hums faintly as it syncs with the station database</span>")
```

Comment:
usr is unsafe in this instance. It's fine for a verb, because verbs are rarely invoked without a usr object. But for procs, definitely should be an argument called `user` or something, and passed properly..